Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+    agent any
+
+    stages {
+
+        stage("Clean") {
+            steps {
+                sh "./gradlew clean"
+            }
+        }
+
+        stage("Assemble") {
+            steps {
+                sh "./gradlew assemble --refresh-dependencies -i"
+            }
+        }
+
+        stage('Publish Amp') {
+            when {
+                anyOf {
+                    branch "master*"
```

Comment:
it's OK to remove the branch constraints while working on this PR (in draft mode), untill you know the publishing actually works

Reply:
ok, will reopen PR once publishing has succeeded